examples/colab/wav2vec2_saved_model_finetuning.ipynb

```diff
@@ -0,0 +1,2441 @@
+{
```

Comment:
<p>This is actually quite interesting. Maybe it's beyond the scope of this colab, but I would expect that blindly truncating like this could negatively impact training. For any samples that are truncated, there is no longer a guarantee that the remaining labels match all the remaining audio, so I would think it's better to discard these samples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>